test/WebJobs.Script.Tests/Microsoft.Azure.WebJobs.Script.WebHost.deps.json

```diff
@@ -29,7 +29,7 @@
   },
   "targets": {
     ".NETCoreApp,Version=v3.1": {
-      "Microsoft.Azure.WebJobs.Script.WebHost/3.9.0": {
```

Comment:
This deps file is out of date, irrespective of my changes. The test is failing in the base branch currently w/o my changes. My PR isn't adding any new dependencies.

Reply:
The key issue seems to be with missing entries here for the dependencies that were flagged.